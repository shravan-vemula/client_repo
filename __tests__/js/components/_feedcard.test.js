import React from 'react';
import {render,cleanup} from '@testing-library/react';
import FeedCard from 'js/_components/molecules/_feedcard';

afterEach(cleanup)

describe("FeedCard Component",()=>{
    it("renders correctly", () => {
        const wrapper = shallow(<FeedCard />);
        expect(wrapper).toMatchSnapshot();
      });

   
});