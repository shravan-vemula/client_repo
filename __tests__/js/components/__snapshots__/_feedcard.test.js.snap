// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedCard Component renders correctly 1`] = `ShallowWrapper {}`;
